src/Adapter/Platform/AbstractPlatform.php

```diff
@@ -19,7 +19,7 @@
     /**
      * @var string
      */
-    protected $quoteIdentifierTo = '\'';
+    protected $quoteIdentifierTo = '""';
```

Comment:
Why would you change the default, this is absolutely a MASSIVE BC BREAK.  I advise you to please no do that especially considering this could have been extended.

Reply:
I could leave it as it is, but in that case I'd comment it, that it's only for BC reasons and contradicts the ANSI SQL standard.
I can't name any SQL implementation that uses single quotes for escaping if identifiers. The single quote is for strings.

I attach a wall of text aka TL;DR:

In chapter `5.2 <token> and <separator>` of 
[Information Technology - Database Language SQL](http://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt) it's defined as

```
         <delimited identifier> ::=
              <double quote> <delimited identifier body> <double quote>

         <delimited identifier body> ::= <delimited identifier part>...

         <delimited identifier part> ::=
                <nondoublequote character>
              | <doublequote symbol>

         <nondoublequote character> ::= !! See the Syntax Rules

         <doublequote symbol> ::= <double quote><double quote>

         <delimiter token> ::=
                <character string literal>
              | <date string>
              | <time string>
              | <timestamp string>
              | <interval string>
              | <delimited identifier>
```

and same is recommended by [Apache docs](https://db.apache.org/derby/docs/10.1/ref/crefsqlj1003454.html):
```
Ordinary identifiers are identifiers not surrounded by double quotation marks. Delimited identifiers are identifiers surrounded by double quotation marks.
```

By standard DB/2, Oracle, PostgreSQL and SQLite use double quote for identifier escaping. Sybase and MS SQLServer default to quoting via [] but for example [SQLServer does support the double quote](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-quoted-identifier-transact-sql) when `SET QUOTED_IDENTIFIER` is `ON`, and it is on, in ANSI mode:
```
When SET ANSI_DEFAULTS is ON, SET QUOTED_IDENTIFIER is enabled.
```

MySQL/MariaDB understand by default only the backtick, but also do support double quote in [ANSI mode](https://dev.mysql.com/doc/refman/5.5/en/compatibility.html):
`SET sql_mode = 'ANSI';` enables it for the running session.